src/helpers/files.ts

```diff
@@ -184,24 +179,16 @@ const EMPTY_STRING = '' as const
 
 const isNegated = (path: string) => path.startsWith('!')
 
-/**
- *
- * @param {string} projectRoot
- * @param {string[]} coverageFilePatterns
- * @returns {Promise<string[]>}
- */
 export async function getCoverageFiles(
   projectRoot: string,
   coverageFilePatterns: string[],
 ): Promise<string[]> {
-  const globstar = (pattern: string) => `**/${pattern}`
-
```

Comment:
These two lines can be removed. The `globstar` variable is shadowing another defined at the top of the file.